packages/katello/katello-repos/katello.repo

```diff
@@ -10,10 +10,10 @@ gpgcheck=@REPO_GPGCHECK@
 # Candlepin RPMs as supported by Katello - This is a coordinated
 # copy of Candlepin's packages in order to ensure compatibility
 
-[katello-candlepin]
+[candlepin]
 name=Candlepin: an open source entitlement management system.
-baseurl=https://yum.theforeman.org/katello/@REPO_VERSION@/candlepin/@DIST@/$basearch/
-gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-foreman
+baseurl=https://yum.theforeman.org/candlepin/@CANDLEPIN_VERSION@/@DIST@/$basearch/
+gpgkey=https://yum.theforeman.org/candlepin/@CANDLEPIN_VERSION@/GPG-RPM-KEY-candlepin
```

Comment:
Copying how Pulpcore is setup in this file right now. There remains the question of how this GPG key ends up here in our newer process with Copr.

I am thinking I may need to add a step that copies the public GPG key into the repository structure during `build_stage_repository` action.

Reply:
Including a file is more reliable. Dnf doesn't check expirations and doesn't refresh if the URL didn't change. I ran into that on Fedora with my personal copr